### PR TITLE
Added support for # comments in the global scope + made it more clear that helper functions are A-OK

### PR DESCRIPTION
**Pull request by eyurko**

### CODE REVIEW

File: client/question/services/code_evaluators/PythonPrereqCheckService.js [hunk 1]
```diff
@@ -107,7 +107,8 @@ tie.factory('PythonPrereqCheckService', [
       for (var i = 0; i < codeLines.length; i++) {
         var line = rightTrim(codeLines[i]);
         var VALID_LINE_PATTERN = /(^\s+)|(^def)|(^import)/;
-        if (line.search(VALID_LINE_PATTERN) === -1 && line !== '') {
+        if (line.search(VALID_LINE_PATTERN) === -1 && line !== '' &&
+            !line.startsWith('#')) {
```
Comment (seanlip) [inline client/question/services/code_evaluators/PythonPrereqCheckService.js]: Think it's worth changing this additional check to "first non-space char on line is a '#'"?

File: client/question/services/FeedbackGeneratorService.js [hunk 2]
```diff
@@ -781,7 +781,8 @@ tie.factory('FeedbackGeneratorService', [
           feedback = FeedbackObjectFactory.create(
             FEEDBACK_CATEGORIES.FAILS_GLOBAL_CODE_CHECK);
           feedback.appendTextParagraph([
-            'Please keep your code within the existing predefined functions',
+            'Please keep your code within the existing predefined functions ',
+            'or define your own helper functions if you need to',
```
Comment (seanlip) [inline client/question/services/FeedbackGeneratorService.js]: Add space at end of this line. Ditto below.

**Comment by rabidbit** [comment]
LGTM

File: client/question/services/code_evaluators/PythonPrereqCheckService.js (hunk 1 above)
Comment (seanlip) [reply client/question/services/code_evaluators/PythonPrereqCheckService.js]: Sorry, ignore this -- thinko on my part. It's caught in VALID_LINE_PATTERN, and those comments would be in a different scope anyway.

**Pull request merged**